internal/provider/tag_resource.go

```diff
@@ -35,6 +38,7 @@ type TagResourceModel struct {
 
 	DigestRef types.String `tfsdk:"digest_ref"`
 	Tag       types.String `tfsdk:"tag"`
+	Tags      []string     `tfsdk:"tags"`
```

Comment:
```suggestion
	Tags      types.List    `tfsdk:"tags"`
```

I'm not sure if this is correct or not.